qb/select.go

```diff
@@ -163,3 +163,44 @@ func (b *SelectBuilder) AllowFiltering() *SelectBuilder {
 	b.allowFiltering = true
 	return b
 }
+
+// Func sets aggregation function for select clause on the query.
+func (b *SelectBuilder) Func(name, column string) *SelectBuilder {
+	b.Columns(name + "(" + column + ")")
+	return b
+}
+
+// Count produces 'count(column)' or 'count(*)'.
+func (b *SelectBuilder) Count(column ...string) *SelectBuilder {
```

Comment:
How about explicitly making `Count(column string)` and `CountAll()` functions.

Reply:
Done.